packages/grafana-ui/src/components/Logs/LogRows.tsx

```diff
@@ -1,12 +1,13 @@
 import React, { PureComponent } from 'react';
-import { cx } from 'emotion';
+import memoizeOne from 'memoize-one';
```

Comment:
Where is this used?

Reply:
I have just reordered imports. It is used here:
```  makeGetRows = memoizeOne((processedRows: LogRowModel[]) => {
    return () => processedRows;
  });``